op/model.py

```diff
@@ -91,6 +91,16 @@ def __init__(self, name, backend, cache):
     def __repr__(self):
         return f'<{type(self).__module__}.{type(self).__name__} {self.name}>'
 
+    @property
+    def is_leader(self):
+        if self.is_local:
+            # This value is not cached as it is not guaranteed to persist for the whole duration
+            # of a hook execution.
+            return self._backend.is_leader()
+        else:
+            # Unable to determine the leadership status for remote units.
```

Comment:
The comment is saying exactly the same thing as the error text below, so it's not adding value.